include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -871,13 +918,17 @@ template<int dim, typename Number>
 std::tuple<unsigned int, unsigned int>
 Operator<dim, Number>::solve_nonlinear(VectorType &       sol,
                                        VectorType const & rhs,
-                                       double const       factor,
+                                       double const       scaling_factor_mass_from_acceleration,
+                                       double const       scaling_factor_mass_from_velocity,
```

Comment:
I would skip `mass_from` in the name (in both cases). The reason is that regarding this function interface, it does not matter whether the damping term is realized as a scaled mass matrix. Imagine we would add another type of damping, then we would need to also adjust the interface of the function, while we only have to change the internal implementation if we use the name `scaling_factor_damping` or `scaling_factor_velocity`.